Basket-Donation-Payments/Basket-Donation-Payments/Models/APIManager.m

```diff
@@ -140,4 +139,41 @@ + (void) newNonprofitConnectedAccountWithEmail:(NSString*)email withAuthorizatio
     [task resume];
 }
 
++ (void)createTransfersWithAmount:(NSNumber*)amount toConnectedStripeAccs:(NSArray<NSString*>*)connectedStripeAccs withSourceTxId:(NSString*)sourceTxId withBlock:(void (^)(NSError *, NSString *))completion {
+    NSInteger numberOfNonprofits = connectedStripeAccs.count;
+    NSInteger amountPerNonprofit = [amount intValue] / numberOfNonprofits;
+    for (NSString *connectedStripeId in connectedStripeAccs) {
+        [self createSingleTransferWithAmount:[NSNumber numberWithLong:amountPerNonprofit] toId:connectedStripeId withSourceTxId:sourceTxId withBlock:completion];
+    }
+}
+
++ (void)createSingleTransferWithAmount:(NSNumber*)amount toId:(NSString*)connectedId withSourceTxId:(NSString*)sourceTxId withBlock:(void (^)(NSError *, NSString *))completion {
+    NSString *backendURL = [APIManager getAPISecretKeysDict][@"Backend_Server_Url"];
+    NSURL *url = [NSURL URLWithString:[NSString stringWithFormat:@"%@create-transfer", backendURL]];
+    NSDictionary *json = @{
+        @"amount": amount,
+        @"currency": @"usd",
+        @"destination": connectedId,
+        @"source_transaction": sourceTxId
+
+    };
+    NSData *body = [NSJSONSerialization dataWithJSONObject:json options:0 error:nil];
+    NSMutableURLRequest *request = [[NSURLRequest requestWithURL:url] mutableCopy];
```

Comment:
Do `[NSMutableURLRequest requestWithURL:url]` instead of `[[NSURLRequest requestWithURL:url] mutableCopy]`, it's more efficient to to create the mutable instance directly than creating an immutable instance first and then a mutable copy.